scripts/install_operator.sh

```diff
@@ -3,23 +3,22 @@
 set -ex
 SCRIPT_DIR=$(dirname "$0")
 SPECS_DIR="$(dirname ${SCRIPT_DIR})/specs"
+OPERATOR_DIR="$(dirname ${SCRIPT_DIR})/kudo-operator"
 
-NAMESPACE=${NAMESPACE:-spark-operator}
-
-OPERATOR_NAME=${OPERATOR_NAME:-spark-operator}
+NAMESPACE=${NAMESPACE:-spark}
 OPERATOR_IMAGE_NAME=${OPERATOR_IMAGE_NAME:-mesosphere/kudo-spark-operator}
 OPERATOR_VERSION=${OPERATOR_VERSION:-latest}
 
 echo "Using namespace '${NAMESPACE}' for installation"
 
-helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+kubectl kudo init || true
 
-if [[ $(helm ls | grep "${OPERATOR_NAME}") ]]; then
-    echo "Spark Operator with name '${OPERATOR_NAME}' already installed"
-    echo "if you want to remove it run: helm delete --purge ${OPERATOR_NAME}"
+if [[ $(kubectl kudo get instances | grep spark) ]]; then
```

Comment:
Hm, I think you said you were able to install multiple instances of spark?

Reply:
Yes, that's right, but for the sake of build automation we currently support only a single instance of Spark Operator. Most likely, it will stay like that because it doesn't seem that Spark Operator namespace isolation works the same way as Mesos Dispatcher group role.

Reply:
Okay, thanks for the additional info